crates/api_common/src/build_response.rs

```diff
@@ -149,10 +149,9 @@ pub async fn send_local_notifs(
     // Get the parent commenter local_user
     let parent_creator_id = parent_comment.creator_id;
 
-    let check_blocks = check_person_instance_community_block(
+    let check_blocks = check_person_community_block(
       person.id,
       parent_creator_id,
-      person.instance_id,
```

Comment:
Maybe this should instead be fixed by passing `community.instance_id`, so that all communities from that instance are ignored but not the users? Seems like that would be consistent with the way instance blocking works in general.

Reply:
I spent a few minutes thinking about it, and I think you're right. Its the community instance_id that should be blocked.